src/error_diffusion.jl

```diff
@@ -149,12 +152,11 @@ $(_error_diffusion_kwargs)
      Scale."  SID 1975, International Symposium Digest of Technical Papers,
      vol 1975m, pp. 36-37.
 """
-function FloydSteinberg(; kwargs...)
-    return ErrorDiffusion(OffsetMatrix([0 0 7; 3 5 1]//16, 0:1, -1:1); kwargs...)
-end
+FloydSteinberg(CT=XYZ; kwargs...) = ErrorDiffusion{CT}(_floydsteinberg_filter; kwargs...)
+const _floydsteinberg_filter = OffsetMatrix([0 0 7; 3 5 1]//16,0:1,-1:1)
```

Comment:
This is very nitpicky feedback, but maybe we could call this `const FLOYDSTEINBERG_FILTER` to be consistent with the constant matrices in `ordered.jl`.